test/fixtures/test-runner/output/arbitrary-output.js

```diff
@@ -0,0 +1,20 @@
+// Flags: --test --expose-internals
+'use strict';
+
+const v8_reporter = require('internal/test_runner/reporter/v8-serializer');
```

Comment:
```suggestion
const v8Reporter = require('internal/test_runner/reporter/v8-serializer');
```

Reply:
if this is the only comment, Il skip it :)